### PR TITLE
Add swagger

### PR DESCRIPTION
**Pull request by miho120**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -28,4 +29,25 @@ def create_app():
...
+    )
+    swagger_spec_prefix = '{}/{}'.format(
+        api_prefix,
+        app.config.get('SWAGGER_SPEC')
+    )
+    swaggerui_blueprint = get_swaggerui_blueprint(
+        swagger_docs_prefix,
+        swagger_spec_prefix,
+        config={
+            'app_name': "Tech API - template"
```
Comment (verdan) [inline app.py]: @miho120 is this needed here? Can we remove this hard-coded string from here ?

File: backend/views.py [hunk 2]
```diff
@@ -25,6 +25,11 @@ def logout():
     oidc.logout()
     redirect_url = request.url_root.strip('/')
     keycloak_issuer = oidc.client_secrets.get('issuer')
-    keycloak_logout_url = '{}/protocol/openid-connect/logout'.format(keycloak_issuer)
-
-    return redirect('{}?redirect_uri={}'.format(keycloak_logout_url, redirect_url))
+    keycloak_logout_url = '{}/protocol/openid-connect/logout'.format(
```
Comment (verdan) [inline backend/views.py:28]: 👍

**Comment by krisss85** [comment]
+1

**Review by verdan — CHANGES_REQUESTED** [review]
Looks good to me. 👍 just a couple of minor comments.

**Review by verdan — APPROVED** [review]

**Pull request merged**